src/model/Siamese_Network.py

```diff
@@ -279,10 +281,10 @@ def evaluate(self, test_file, batch_size, analyze=False):
     def predict(self, x_test, names):
         prob = self.siamese_net.predict(x_test)
 
-        for pair_index in range(len(names)):
-            name = names[pair_index]
-            pair_prob = prob[pair_index][0]
-            print(f"Similar to {name} with probability: ", pair_prob)
+        name = names[0]

```

Comment:
Ideally should use for loop to loop over all elements, rather than predicting only on one element